views/components/title-block.jsx

```diff
@@ -0,0 +1,12 @@
+import { createElement } from "complate-stream";
+import Container from "./container";
+import Logo from "./logo";
+
+export default (_params, ...children) => <div class="title-block">
+	<Container>
+		<Logo src="complate_device_static-512.png" />
```

Comment:
```suggestion
		<Logo src="complate.png" />
```

(no need to retain the original, awkward file name)

Also, that image file is huge: 95 kB - we should run it through [faucet-images](https://www.faucet-pipeline.org/static#compact).

Reply:
I've updated the filename and run it through faucet-images to get a smaller file size.